pkg/platform/runtime/runtime.go

```diff
@@ -193,6 +194,8 @@ func (r *Runtime) recordCompletion(err error) {
 		errorType = "solve"
 	case errs.Matches(err, &setup.BuildError{}) || errs.Matches(err, &buildlog.BuildError{}):
 		errorType = "build"
+	case errs.Matches(err, bpModel.BuildPlannerError{}):
```

Comment:
Does this need to be a pointer?

Reply:
Nice catch! Technically I believe it should work regardless, but better safe than sorry.